source/view-model.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Created by Ksenia Annenko on 24.02.2017.
+ */
+
+/*********************** View-Model **********************/
+"use strict";
+function ViewModel() {
+    this.init = function () {
+        let firstTestPlayer = 'First';
+        let secondTestPlayer = 'Second';
+        this.game = new Game(false, firstTestPlayer, secondTestPlayer);
+        this.gameInfo = new GameInfo(this.game);
+        this.game.startTurn();
+        this.finishTurn = function (id) {
+            this.game.finishTurn(id);
+        };
+    };
+
+    this.getCurrentPlayer = function () {
+        return  this.gameInfo.getCurrentPlayer().playerName;
+    };
+
+    this.getCurrentStatus = function () {
```

Comment:
Can we move this logic into GameInfo and use it everywhere where this function is used in the view?